src/Avalonia.Controls/Automation/AutomationProperties.cs

```diff
@@ -0,0 +1,539 @@
+using System;
+using Avalonia.Controls;
+
+namespace Avalonia.Automation
+{
+    public static class AutomationProperties
```

Comment:
A `HeadingLevel` property is missing, however it makes scanning documents a lot easier for people using assistive technology. Also, a `LandMarkType` property is missing but is very important for people with assistive technology.

Reply:
This was ported from WPF which doesn't have `HeadingLevel`/`LandmarkType` . I can add those.

Reply:
Those properties are present in UWP and the web (as semantic html). Honestly, I'm a bit surprised that WPF didn't have those.

Reply:
So should we add the `HeadingLevel` or `LandMarkType` properties? When using narrator on Windows, those are definitely helping people navigate UWP/WinUI apps more easily and those properties are also present in web accessibility.

Reply:
Yeah definitely, as I mention in the PR description though, this PR is already huge so stuff like this will be implemented later.